### ISSUE TITLE
Train fine, but merge / convert not

### ISSUE THREAD

**Issue by kalmsy** [deepfakes/faceswap#69]
Please can anyone determine why this issue may be occurring when trained to very low amount (<0.02) , preview window looking fine but at convert / merge stage it turns into a mess?

Please forgive if this is an intrusion as strictly this is concerned with reddit.com/r/FakeApp, but i noticed user Clorr1's offer of helping out and pointing to this repo, which presumably fakeApp implements. I have seen several people with the issue on /r/fakeapp & r/deepfakes

[screenshot](https://imgur.com/a/ACcq2)
[screenshot](https://i.imgur.com/gsyNcp1.png)
[screenshot](https://imgur.com/a/QRpqU)

It's really frustrating because the training seems to be fine in the earlier stages (from early merge test runs) but once things start getting accurate in the preview window merge (now convert) turns the whole thing into zombie mode. So far i can't see anything in common between users like myself who have this issue. I have tried a lot of different troubleshooting steps but they are random tbh.

Downgrading graphics driver to one shipped with CUDA 8.0
switching A and B to match file format
running app in admin mode
uninstalling coincidentally installed python

I'm really sure about the quality of the celeb images there is nothing to indicate they are problematic, after all why does it start fine and end like crap with the preview window showing good results?

Thanks for any help

Nvidia 1070
Win 10

**Comment by Clorr** [comment]
I don't know much about this issue, but I think it comes from a bad training set of images. Please note that if you have too few images or too few diversity in images, the model cannot invent angles he does not know about. If you think it is not the case, feel free to reopen

**Comment by Mattbilo** [comment]
Did you find a solution to this problem with FakeApp?

I'm having the same issue, previews during training look great with loss less than a 0.01, but the merged/converted pics look like a monster. 

Loss monitoring: https://i.imgur.com/QwJO4tG.png
Previews: https://i.imgur.com/xz65Fus.png
Converted (zoomed in): https://i.imgur.com/vulbdt0.png

**Comment by Mattbilo** [comment]
I'm using 250 frames extracted from a very short video for A, with subject A looking only straight forward and down, to keep things simple for a first test.

I'm using 1500 frames extracted from a long video for B (5000 original frames, of which I removed 3500 that were not matching orientation or expressions of the 250 in video B).

Each and every single input A is converting well in the previews, but not in the merged result.

Example: https://i.imgur.com/4cG3MWb.png

NOTE: I think it's an issue with FakeApp 1.1. Since my first post above, I tried using the open-sourced OpenFaceSwap, and results are like the previews :)))
